package.json

```diff
@@ -16,6 +16,8 @@
   },
   "scripts": {
     "precommit": "yarn lint && yarn unit-test && yarn int-test",
+    "pre-commit-lite": "yarn lint && yarn unit-test",
```

Comment:
Left existing precommit command in case anyone is using it

Reply:
LGTM